docs/framework_documentation/abstract_level/entity_types/group.rst

```diff
@@ -33,25 +33,30 @@ Basic relationships to other entity-types
 
 A group will usually...
 
--  have several member :doc:`individuals<individual>`
+-  have several members :doc:`individuals<individual>`,
+   which is represented by a group membership directed network owned by the culture taxon
 
 In addition, a group may...
 
--  have one or several "leader" :doc:`individuals<individual>`, 
-   of which one may be the dominant leader
+-  have an "intra" group network between members :doc:`individuals<individual>`
+   (this feature is not completely implemented yet)
```

Comment:
only describe (and explicitly) what feature is implemented and add `TODO` tags in the code.